src/components/Book.js

```diff
@@ -46,19 +44,21 @@ const Book = (props) => {
               </div>
             </div>
             <div className="percent">
-              <span className="circle" style={progress}> </span>
-              <div>
+              <span className="circle" style={progress}>
+                <div className="inner"> </div>
+              </span>
+              <div className="progress-contain">
```

Comment:
- [ ] I believe the progress percentage is darker in color. Please refer to the design given.
- [ ] I would also urge to you reduce its font-size as well